PostHog/PostHogRemoteConfig.swift

```diff
@@ -74,17 +74,25 @@ class PostHogRemoteConfig {
         }) {
             dispatchQueue.async {
                 self.reloadRemoteConfig { [weak self] remoteConfig in
-                    guard let self else { return }
-
-                    let hasFeatureFlags = remoteConfig?[self.hasFeatureFlagsKey] as? Bool == true
+                    // if there's no remote config response, skip
+                    guard let self, let remoteConfig else { return }
```

Comment:
i think we should call `notifyFeatureFlags(nil)`

Reply:
https://github.com/PostHog/posthog-ios/blob/784145ddd2f7de3a6d6fcce0e50c04f29d33b338/PostHog/PostHogRemoteConfig.swift#L159-L161
is another case of logging + calling notifyFeatureFlags(nil)